amplipi/updater/asgi.py

```diff
@@ -95,6 +98,80 @@ class ReleaseInfo(BaseModel):
   logger.exception(f'Error loading identity file: {e}')
 
 
+@router.get("/settings/persist_logs")
+def get_log_persist_state():
+  """
+  Checks /etc/systemd/journald.conf to find if the current storage setting is persistent and returns a bool
+  Note that returning false doesn't necessarily mean that logs are set to volatile, and could just mean that the config file is missing the line being read
+  """
+  conf_path = '/etc/systemd'
+  config = configparser.ConfigParser(strict=False, allow_no_value=True)
+  config.read(f'{conf_path}/journald.conf')
+  return config.get("Journal", "Storage", fallback="") == "persistent"
+
+
+@router.post("/settings/persist_logs")
+def toggle_persist_logs(value: Optional[bool] = None):
```

Comment:
`value` seems to be unused

Reply:
It's only really used by the delete endpoint, I've also tried to use it with the frontend to catch an edgecase but that proved unnecessary and error prone